pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -151,6 +154,7 @@ func TestNSMGR_RemoteUsecase_BusyEndpoints(t *testing.T) {
 
 	defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*5)
+	ctx = log.WithLog(ctx)
```

Comment:
I thought we discussed this moment here: 
https://github.com/networkservicemesh/sdk/pull/641#discussion_r560802505